api/config/module/proxy/v1/proxy.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+
+package clutch.config.module.passthrough.v1;
+
+option go_package = "github.com/lyft/clutch/backend/api/config/module/passthrough/v1;passthroughv1";
```

Comment:
Should this be proxy instead of passthrough?

Reply:
Yup I think we raced on my commit